styles/base.css

```diff
@@ -19,7 +19,7 @@
   --f-serif: 'Noto Serif', Georgia, serif;
 
   /* Dimensions */
-  --d-max-layout-width: 1180px;
```

Comment:
Adjusted the layout so it's ~1200 pixels (accounting for the 30px of horizontal padding).